flexmeasures/data/services/scheduling.py

```diff
@@ -140,9 +176,17 @@ def make_schedule(
         flex_model=flex_model,
         flex_context=flex_context,
     )
+
+    if is_fallback:
+        scheduler: Scheduler | None = scheduler.fallback_scheduler
```

Comment:
What is happening here?

Reply:
The code changed too much... and this line is gone.